test/debugger/test-debugger-client.js

```diff
@@ -25,40 +25,40 @@ p.execute('Type: connect\r\n' +
           'Protocol-Version: 1\r\n' +
           'Embedding-Host: node v0.3.3-pre\r\n' +
           'Content-Length: 0\r\n\r\n');
-assert.equal(1, resCount);
+assert.strictEqual(1, resCount);
```

Comment:
Nit: can you please swap the arguments here and throughout the rest of the file where it makes sense?
```js
assert.strictEqual(resCount, 1);
```
This because the first argument is the `actual` and the second is the `expected`.
Feel free to ignore this :)